<ui version="4.0" >
 <class>LoginDialogBase</class>
 <widget class="QDialog" name="LoginDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Maestro Client Login</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="MaestroResource.qrc" >:/Maestro/images/maestro_icon.png</iconset>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="mLockLbl" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="MaestroResource.qrc" >:/Maestro/images/lock.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mLoginLbl" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Maestro Login</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="mPasswordEdit" >
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="mUserLbl" >
       <property name="text" >
        <string>User:</string>
       </property>
       <property name="buddy" >
        <cstring>mUserEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="mDomainLbl" >
       <property name="text" >
        <string>Domain:</string>
       </property>
       <property name="buddy" >
        <cstring>mDomainCB</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QComboBox" name="mDomainCB" >
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="mUserEdit" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="mPasswordLbl" >
       <property name="text" >
        <string>Password:</string>
       </property>
       <property name="buddy" >
        <cstring>mPasswordEdit</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mLoginBtn" >
       <property name="text" >
        <string>&amp;Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancelBtn" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mUserEdit</tabstop>
  <tabstop>mPasswordEdit</tabstop>
  <tabstop>mDomainCB</tabstop>
  <tabstop>mLoginBtn</tabstop>
  <tabstop>mCancelBtn</tabstop>
 </tabstops>
 <resources>
  <include location="MaestroResource.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>mLoginBtn</sender>
   <signal>clicked()</signal>
   <receiver>LoginDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>272</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel" >
     <x>92</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>LoginDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>355</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel" >
     <x>175</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mUserEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>LoginDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>104</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel" >
     <x>104</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mPasswordEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>LoginDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>233</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel" >
     <x>136</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
